src/components/container/ReasonsSection/ReasonsSection.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+
+import skillBasedLogo from "../ReasonCard/assets/icons/skill-based-search.svg";
+import continousUpskillingLogo from "../ReasonCard/assets/icons/continous-upskilling.svg";
+import bootcampsLogo from "../ReasonCard/assets/icons/bootcamps.svg";
+import widerReachLogo from "../ReasonCard/assets/icons/wider-reach.svg";
+import easyApplicationLogo from "../ReasonCard/assets/icons/easy-application-process.svg";
+import trainingDevelopmentLogo from "../ReasonCard/assets/icons/training-development.svg";
+
+import { staticReasons } from "../../../state/models/staticReasons";
+import ReasonCard from "../ReasonCard/ReasonCard";
+import "./ReasonsSection.scss";
+
+function ReasonsSection() {
+  const pathToLogo = {
+    "skill-based-search": skillBasedLogo,
+    "continous-upskilling": continousUpskillingLogo,
+    bootcamps: bootcampsLogo,
```

Comment:
`bootcamps` property should also have quotation marks

Reply:
done